packages/cubejs-schema-compiler/src/adapter/BaseQuery.js

```diff
@@ -3857,6 +3857,11 @@ export class BaseQuery {
    * @return {string}
    */
   dimensionTimeGroupedColumn(dimension, granularity) {
+    // Handle case when granularity is not specified (e.g., time dimension used only for filtering)
+    if (!granularity) {
```

Comment:
Can you show the example when this function is called without `granularity` object?